modules/core/help.py

```
@@ -0,0 +1,295 @@
+@hlp.command('<module> [--legacy] {{core.help.help.detail}}',
+             options_desc={'--legacy': '{help.option.legacy}'})
+async def bot_help(msg: Bot.MessageSession, module: str):
+    module_list = ModulesManager.return_modules_list(
+        target_from=msg.target.target_from)
+    alias = ModulesManager.modules_aliases
+    if msg.parsed_msg:
+        msgs = []
+        if module in alias:
+            help_name = alias[module].split()[0]
+        else:
+            help_name = module.split()[0]
```

Ruff

Use ternary operator `help_name = alias[module].split()[0] if module in alias else module.split()[0]` instead of `if`-`else`-block

Replace `if`-`else`-block with `help_name = alias[module].split()[0] if module in alias else module.split()[0]`

(SIM108)